src/App.jsx

```diff
@@ -18,6 +18,7 @@ import AdminNavMenu from './components/navigation/AdminNavMenu';
 import AdminDashboard from './pages/AdminDashboard';
 import EventSignUp from './pages/EventSignUp';
 import VolunteerDashboard from './pages/VolunteerDashboard';
+import VolunteeringHistory from './components/profile/VolunteeringHistory';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'VolunteeringHistory' is defined but never used.